holmes/config.py

```diff
@@ -211,6 +213,32 @@ class Config(RobustaBaseConfig):
 
     _server_tool_executor: Optional[ToolExecutor] = None
 
+    version: Optional[str] = Field(
```

Comment:
What is the reason this is in the config?
Is it something the user can configure?